android/src/main/kotlin/com/hypertrack/sdk/flutter/HyperTrackPlugin.kt

```diff
@@ -77,7 +80,7 @@ public class HyperTrackPlugin(): FlutterPlugin, MethodCallHandler, StreamHandler
 
     val sdk = sdkInstance
     if (sdk == null) {
-      result.error("NOT_INITIALIZED", "Sdk wasn't initialized", null)
+      result.error("NOT_INITIALIZED", "Internal Error: onMethodCall(@NonNull call: MethodCall, @NonNull result: Result) - sdkInstance is null", null)
```

Comment:
you can omit the types from the method name
```suggestion
      result.error("NOT_INITIALIZED", "Internal Error: onMethodCall(call:,result:) - sdkInstance is null", null)
```

Reply:
Why do we need to log param names without the values?
I think it should be "Internal Error: onMethodCall(${call.method}) - sdkInstance is null"

Reply:
it's to designate the method that failed, which is the topLvl `onMethodCall(call:,result:)` -- we could also pass method call as param, but I'm not sure if it's going to read well in the error log. I also think it doesn't matter what the method call was in this case, because the `sdkInstance` should not be null.

Reply:
edit: let's pass the `call.method` in the error log @AshishSynoriq